content/20.results.md

```diff
@@ -13,22 +13,31 @@ We scraped PubMed for manuscripts written in English from 1993--2019 with the Me
 We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 176,110 articles for evaluation.
 For each article, we extracted its last author's fore name and last name for analysis.
 
-### Evaluation Set and Field-specific Background
+### Defining metrics and a field-specific background
 
-Defining an appropriate background distribution is key to determining the extent to which representation diverges from that background.
-We were faced with multiple decisions as we sought to define a field-specific distribution.
-With insightful feedback from the reviewers, after multiple iterations of the analysis (see Methods), we extended the analysis to all 176,110 computational biology PubMed articles, extracted names of last authors, and analyzed the raw prediction probability values to capture the uncertainty of the prediction model. (Fig. {@fig:workflow}).
-We assumed that, among the authors of a specific paper, the last author (often research advisors) would be most likely to be invited for a keynote or honored as a fellow, and so comparing honors to last authorships would be appropriate.
+We were faced with a number of choices as we set out to examine representation within the field.
+Here, we outline the choices that we made and the rationale for these choices.
 
-This study focuses on the field of computational biology.
-However, our analysis framework can be adapted to other fields if the researcher curates their own set of honors and replaces the MeSH term "computational biology" with their topic of interest.
-While PubMed contains the most comprehensive resource of biomedical texts and associated standardized metadata of which we are aware, a difference database like [arXiv](http://arxiv.org/) may be more appropriate for extracting articles in other fields.
-
-The units in this analysis are honors and authorships, not individuals.
+For one step, we needed to determine whether the most appropriate unit for analysis was researchers or honor and authorship events.
+We elected to perform the analysis at the level of honors and authorships, not individuals.
+Our rationale was that, if a scientist was honored three times, these three honoree slots represented distinct selection processes and should be considered separately.
+We also elected to use authorships instead of authors because name disambiguation can be error prone, and because the overall design of the work was to examine distributions the precise linking of researchers was not necessary to address the research question.
```

Comment:
```suggestion
We also elected to use authorships instead of authors because name disambiguation can be error prone, and because the overall design of the work was to examine authorship distributions, the precise linking of researchers was not necessary to address the research question.
```